app/src/main/res/values/strings.xml

```diff
@@ -38,6 +38,25 @@
     <string name="forgot_pass_fragment_button_reset_password">Reset password</string>
     <string name="forgot_pass_fragment_button_verify">Verify</string>
 
+    <!-- VEHICLE VIEW STRINGS -->
+    <string name="home_fragment_2d_mode">2D</string>
+    <string name="home_fragment_3d_mode">3D</string>
+
+    <!-- VEHICLE COMPONENTS STRINGS -->
+    <string name="bottom_sheet_title">Parameters</string>
+    <string name="bottom_sheet_title_2">Information</string>
```

Comment:
This is a little to abstract. Name them something like bottom_sheet_title_information and _parameters